src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrScanOptions.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommandLine;
+
+#nullable enable
+namespace Microsoft.DotNet.Darc.Options.VirtualMonoRepo;
+
+internal abstract class VmrScanOptions : VmrCommandLineOptions
+{
+    [Option("baseline-file", Required = true, HelpText = "Path to the file containing a list of the VMR baseline files")]
+    public string BaselineFilesPath { get; set; } = string.Empty;
```

Comment:
I think it's just one baseline, one file, but contains a list of files. The list is the baseline. So `BaselineFilePath` makes more sense?

```suggestion
    public string BaselineFilePath { get; set; } = string.Empty;
```

Reply:
Applies to the other variable names in the PR